orchagent/vxlanorch.h

```diff
@@ -148,18 +237,17 @@ class VxlanTunnelRequest : public Request
 
 typedef std::unique_ptr<VxlanTunnel> VxlanTunnel_T;
 typedef std::map<std::string, VxlanTunnel_T> VxlanTunnelTable;
-
-typedef enum
-{
-    TUNNEL_MAP_T_VLAN,
-    TUNNEL_MAP_T_BRIDGE,
-    TUNNEL_MAP_T_VIRTUAL_ROUTER,
-} tunnel_map_type_t;
+typedef std::map<uint32_t, uint16_t> VxlanVniVlanMapTable;
+typedef std::map<IpAddress, VxlanTunnel*> VTEPTable;
 
 class VxlanTunnelOrch : public Orch2
 {
 public:
-    VxlanTunnelOrch(DBConnector *db, const std::string& tableName) : Orch2(db, tableName, request_) { }
+    VxlanTunnelOrch(DBConnector *statedb, DBConnector *db, const std::string& tableName) :
```

Comment:
This change will impact the `orchdaemon.cpp` 
line 142 `VxlanTunnelMapOrch *vxlan_tunnel_map_orch = new VxlanTunnelMapOrch(m_applDb, APP_VXLAN_TUNNEL_MAP_TABLE_NAME);`

Reply:
will change orchdaemon.cpp